app/src/main/java/com/github/factotum_sdp/factotum/placeholder/DestinationRecords.kt

```diff
@@ -2,18 +2,26 @@ package com.github.factotum_sdp.factotum.placeholder
 
 import com.github.factotum_sdp.factotum.data.Action
 import com.github.factotum_sdp.factotum.data.DestinationRecord
+import java.text.SimpleDateFormat
+import java.util.*
 
+/**
+ * Temporary PlaceHolder for some DestinationRecord synthetic data
+ *
+ * @see DestinationRecord
+ */
 object DestinationRecords {
 
     val RECORDS: MutableList<DestinationRecord> = ArrayList()
-    val RECORD_TO_ADD: DestinationRecord = DestinationRecord(null, "new", 2, arrayListOf(Action.PICK))
-    val RECORD_LAST: DestinationRecord = DestinationRecord(null, "last", 2, arrayListOf(Action.PICK))
+    val RECORD_TO_ADD: DestinationRecord = DestinationRecord("new", null,0, 2, arrayListOf(Action.PICK))
+    val RECORD_LAST: DestinationRecord = DestinationRecord("last, ", null,0, 2, arrayListOf(Action.PICK))
     init {
```

Comment:
Might be a tiny bit cleaner to replace all of the .add with an .addAll. Probably will take care of code climate complaining as well.